packages/babel-plugin-transform-react-jsx/test/fixtures/react/handle-spread-with-proto/output.js

```diff
@@ -0,0 +1,19 @@
+var __proto__ = null;
+
+/*#__PURE__*/
+React.createElement("p", {
+  __proto__: null
+}, "text");
+
+/*#__PURE__*/
+React.createElement("div", {
+  "__proto__": null
+}, contents);
```

Comment:
Should `spreadElement` be used here?
It seems to be the expected behavior in https://github.com/babel/babel/issues/14756#issue-1304441362.

Reply:
Good catch. Yeah the spread element should not be stripped. I will investigate.